tests/plugins/clipboard/paste.js

```diff
@@ -808,6 +808,27 @@
 			} );
 		},
 
+		// (#1321)
+		'htmlified text unification 7 - IDEOGRAPHIC space': function() {
```

Comment:
This test seems possible to be simplified. Cases between browsers are very similar.